apps/expo-go/android/expoview/build.gradle

```diff
@@ -194,34 +190,20 @@ dependencies {
   releaseApi 'com.squareup.leakcanary:leakcanary-android-no-op:1.4-beta1'
   api 'commons-io:commons-io:2.6'
   api 'me.leolin:ShortcutBadger:1.1.4@aar'
-  implementation "com.github.CanHub:Android-Image-Cropper:4.3.0"
+  implementation "com.github.CanHub:Android-Image-Cropper:4.3.1"
   api 'commons-codec:commons-codec:1.10'
   api 'net.openid:appauth:0.7.1'
   api 'com.airbnb.android:lottie:3.4.0'
   compileOnly 'io.branch.sdk.android:library:4.1.0'
   api "androidx.exifinterface:exifinterface:1.0.0"
-  api "androidx.legacy:legacy-support-v4:1.0.0"
   api "androidx.browser:browser:1.0.0"
   implementation "androidx.tracing:tracing-ktx:1.1.0"
 
   // expo-face-detector
   implementation 'com.google.mlkit:face-detection:16.1.5'
 
-  // expo-ads-admob
-  implementation 'com.google.android.gms:play-services-ads:20.5.0'
-  // For apps targeting Android 12, add WorkManager dependency.
-  // See more at https://developers.google.com/admob/android/rel-notes
-  constraints {
-    implementation('androidx.work:work-runtime:2.7.0') {
-      because '''androidx.work:work-runtime:2.1.0 pulled from
-      play-services-ads has a bug using PendingIntent without
-      FLAG_IMMUTABLE or FLAG_MUTABLE and will fail in Apps
-      targeting S+.'''
-    }
-  }
-
-  api 'com.google.firebase:firebase-core:21.1.0'
-  api 'com.google.firebase:firebase-messaging:22.0.0'
```

Comment:
we may also remove dep at https://github.com/expo/expo/blob/main/apps/expo-go/android/expoview/build.gradle#L236-L237 below